releases/release-1.10/release-notes-draft.md

```diff
@@ -1,3 +1,1018 @@
+
+* The kubelet  --docker-disable-shared-pid flag, which runs docker containers with a process namespace that is shared between all containers in a pod, is now deprecated and will be removed in a future release. It is replaced by `v1.Pod.Spec.ShareProcessNamespace`, which configures this behavior. This field is alpha and can be enabled with --feature-gates=PodShareProcessNamespace=true. ([#58093](https://github.com/kubernetes/kubernetes/pull/58093), [@verb]([https://github.com/verb](https://github.com/verb)))
+
+* The kubelet's cadvisor port has been deprecated. The default will change to 0 (disabled) in 1.12, and the cadvisor port will be removed entirely in 1.13. ([#59827](https://github.com/kubernetes/kubernetes/pull/59827), [@dashpole](https://github.com/dashpole))
+
+* rktnetes has been deprecated in favor of rktlet. Please see [https://github.com/kubernetes-incubator/rktlet](https://github.com/kubernetes-incubator/rktlet) for more information. ([#58418](https://github.com/kubernetes/kubernetes/pull/58418), [@yujuhong](https://github.com/yujuhong))
+
+* The Kubelet now explicitly registers all of its command-line flags with an internal flagset, which prevents flags from third party libraries from unintentionally leaking into the Kubelet's command-line API. Many unintentionally leaked flags are now marked deprecated, so that users have a chance to migrate away from them before they are removed. In addition, one previously leaked flag, --cloud-provider-gce-lb-src-cidrs, has been entirely removed from the Kubelet's command-line API, because it is irrelevant to Kubelet operation. The deprecated flags are: 
+
+--application_metrics_count_limit
+--boot_id_file
+--container_hints
+--containerd
+--docker
+--docker_env_metadata_whitelist
+--docker_only
+--docker-tls
+--docker-tls-ca
+--docker-tls-cert
+--docker-tls-key
+--enable_load_reader
+--event_storage_age_limit
+--event_storage_event_limit
+--global_housekeeping_interval
+--google-json-key
+--log_cadvisor_usage
+--machine_id_file
+--storage_driver_user
+--storage_driver_password
+--storage_driver_host
+--storage_driver_db
+--storage_driver_table
+--storage_driver_secure
+--storage_driver_buffer_duration
+
+([#57613](https://github.com/kubernetes/kubernetes/pull/57613), [@mtaufen]([https://github.com/mtaufen](https://github.com/mtaufen)))
+
+* The boostrapped RBAC role and rolebinding for the `cloud-provider` service account is now deprecated. If you're currently using this service account, you must create and apply your own [RBAC polic](https://kubernetes.io/docs/admin/authorization/rbac/)[y](https://kubernetes.io/docs/admin/authorization/rbac/) for new clusters. ([#59949](https://github.com/kubernetes/kubernetes/pull/59949), [@nicksardo]([https://github.com/nicksardo](https://github.com/nicksardo)))
```

Comment:
weird link split "polic" and "y" are linked separately

Reply:
Fixed.